lesson6.html

```diff
@@ -106,22 +108,31 @@ <h3>Dictionary operations</h3>
         <section>
           <h3>Dictionary exercise</h3>
 
-          (⭐️⭐️) 6a. Let's go shopping!
+          (⭐️⭐️) 6a. Let's access data in a dictionary!
           <pre><code class="python" data-trim>
-              inventory = {'apples': 3, 'pears': 2, 'bananas': 0}
+            hotel = {
+                'name': 'Corinthia Hotel',
+                'amenities': ['Indoor pool', 'Fitness centre', 'Spa'],
+                'top spots': [
+                    {'name': 'Trafalgar Square',
+                     'proximity': 300},
+                    {'name': 'London Eye',
+                     'proximity': 450},
+                    {'name': 'Buckingham Palace',
+                     'proximity': 1200}
+                 ]
+            }
           </code></pre>
-          <br>
-          The program should:
-          <ol>
-            <li>
-              Ask what the shopper would like to buy, and take user input.
-            </li>
-            <li>
-              Minus 1 of that item from that inventory and print the new inventory.
-              <br>
-              Assume that the user will type 'apples', 'pears' or 'bananas'!
-            </li>
-          </ol>
+                    
+          <small>
+            How can you:<br>
+            <ul>
+              <li>Access the hotel name?</li>
+              <li>Find out how many amenities it has?</li>
+              <li>Find out the names of the top spots near it?</li>
+              <li>Find out all the top spots within 500 metres reach?</li>>
```

Comment:
There's a double closing angular bracket at the end

Reply:
60f940bde0ea055c382415ecac4d3395ce38bff1